provider/qemu_img_utils.py

```diff
@@ -115,3 +115,32 @@ def check_flag(strace_log, target_file, flag):
         for line in lines:
             logging.debug(line.strip())
         return any(flag in line for line in lines)
+
+
+def setup_loop_dev(export_img):
+    """
+    Setup a loop device with a file image.
+
+    :param export_img: The image file used to setup loop device
+    :return: The loop device under /dev
+    """
+    try:
+        process.run("losetup -f %s" % export_img)
+    except process.CmdError as err:
+        raise avocado.TestError("Command line failed: %s" % err)
+    target = process.system_output(
+        "losetup -l | grep %s | awk '{print $1}'" % export_img,
+        shell=True).decode()
+    return target
+
+
+def free_loop_dev(loop_dev):
+    """
+    Free a loop device.
+
+    :param loop_dev: The loop device will be free
+    """
+    try:
```

Comment:
same as above, no need to raise a `avocado.TestError`

Reply:
Updated, thanks.